flow/workflows/cdc_flow.go

```diff
@@ -570,6 +571,11 @@ func CDCFlowWorkflow(
 				return nil, err
 			}
 
+			if finishedError {
```

Comment:
this seems weak, since the only way to hit this with success is pause/resume, editing mirror, or waiting an absurdly long time